### ISSUE TITLE
MultiPolygon not supported by OnFeatureClickListener?

### ISSUE THREAD

**Issue by migrations** [googlemaps/android-maps-utils#320]
![screenshot_2_polygons](https://cloud.githubusercontent.com/assets/5394572/19514240/22de18ca-95f3-11e6-9d65-affafaad7018.JPG)
**Summary:** 
When clicking on geojson from type MultiPolygon then setOnFeatureClickListener receives not feature and crashes when you want to use a getProperty 

**Steps to reproduce:** 

Use following JSON in the GeoJsonDemoActivity.

replace for example the content of raw/earthquakes.json with this:

```
{
  "type": "FeatureCollection",
  "features": [{
    "type": "Feature",
    "geometry": {
      "type": "MultiPolygon",
      "coordinates": [
        [
          [
            [25, 41],
            [25, 42],
            [26, 42],
            [26, 41],
            [25, 41]
          ]
        ]
      ]
    },
    "properties": {
      "title": "Test Multipolygon"
    }
  }, {
    "type": "Feature",
    "geometry": {
      "type": "Polygon",
      "coordinates": [
        [
          [27, 41],
          [27, 42],
          [28, 42],
          [28, 41],
          [27, 41]
        ]
      ]
    },
    "properties": {
      "title": "Test Polygon"
    }
  }]
}

```

ps: i've worked around this issue by splitting all MultiPolygons into Polygons in the app, but that is a "little" ugly ;)

**Comment by NativeDevlopment** [comment]
Hello Team,
I am facing same issue, could you please suggest me, how to resolve this issue.

Thanks
Amar

**Comment by Libby713** [comment]
Fixed in  f848679

**Comment by jayraghav** [comment]
Hello Team ,
i am unable  to get different different polygon properties on Click of Multy polygon click listener. , could you suggest me how to resolve this issue,
layer.setOnFeatureClickListener(new GeoJsonLayer.GeoJsonOnFeatureClickListener() {
    @Override
    public void onFeatureClick(GeoJsonFeature geoJsonFeature) {
        if(geoJsonFeature.hasProperty("id")){
            Log.e("get feature",geoJsonFeature.getPropertyKeys().toString());
            Log.e("id",geoJsonFeature.getProperty("id"));
            Log.e("Name",geoJsonFeature.getProperty("Name"));
        }

    }
});
Thanks
Amar
